etc/profile-m-z/megaglest.profile

```diff
@@ -19,7 +21,7 @@ include disable-xdg.inc
 mkdir ${HOME}/.megaglest
 whitelist ${HOME}/.megaglest
 whitelist /usr/share/megaglest
-whitelist /usr/share/games/megaglest	# Debian version
+whitelist /usr/share/games/megaglest # Debian version
```

Comment:
It's common to separate same-line comments with more than one space.

Reply:
[PEP 8](https://peps.python.org/pep-0008/) for example

> An inline comment is a comment on the same line as a statement. Inline comments should be separated by at least two spaces from the statement. They should start with a # and a single space.

Reply:
It actually looks like it was even a tab. In any case, I did a quick regex grep and it would be the only comment after a directive, so I moved it to a dedicated line.